app/Http/routes.php

```diff
@@ -42,6 +42,7 @@
     Route::get('messages/{message}/send', 'MessagesController@sendMessage')->name('messages.send');
 
     // Users
+    Route::get('users/contestants', 'UsersController@contestantIndex')->name('users.contestants');
```

Comment:
before someone says something about this being above the resource, see what I wrote in #context